srcOA3/mixins/response/absoluteTest.yml.njk

```diff
@@ -0,0 +1,6 @@
+application/json:
```

Comment:
I had re-used an existing mixin to test the absolute paths worked, but got completely confused by the json-schema-ref-parser de-duping the output with complex references.

So, the test case was easier to parse if I used a separate mixin.